code/src/Number.js

```diff
@@ -0,0 +1,44 @@
+import React from "react"
+
+const numberRanges = [
+    "0-5",
+    "6-10",
+    "10-15",
+    "16-20",
+    ">20"
+]
+
+export const Number = ({ userNumber, onNumberOfBooksChange }) => {
+    return (
+        <>
+            <section className="question white">
```

Comment:
i think the className is missing a - ? 
So "question-white" maybe? :)

Reply:
It's two different classes that I apply, that's why there is no - between.